src/sentry/ingest/ingest_consumer.py

```diff
@@ -313,7 +313,8 @@ def dispatch_task(cache_key: str) -> None:
 
 
 def _store_event(data) -> str:
-    return event_processing_store.store(data)
+    with metrics.timer("ingest_consumer._store_event"):
```

Comment:
Any extra tags you want to set on this metric, like for example `data.get("type")`?